sql/mob_pools.sql

```diff
@@ -6125,14 +6125,14 @@ INSERT INTO `mob_pools` VALUES (6082,'Steelshank_Kratzvatz','Steelshank_Kratzvat
 INSERT INTO `mob_pools` VALUES (6083,'Kratzvatzs_Hecteyes','Kratzvatzs_Hecteyes',139,0x0000800100000000000000000000000000000000,4,4,12,280,100,0,1,0,1,0,0,32,865,155,0,0,10,0,0,139);
 INSERT INTO `mob_pools` VALUES (6084,'Bloodfist_Voshgrosh','Bloodfist_Voshgrosh',334,0x0000160400000000000000000000000000000000,8,2,8,240,100,0,1,1,1,2,0,32,7,159,0,0,5,0,0,334);
 INSERT INTO `mob_pools` VALUES (6085,'Spellspear_Djokvukk','Spellspear_Djokvukk',334,0x0000180400000000000000000000000000000000,14,5,8,240,100,0,1,1,1,2,0,32,1195,159,0,0,3,0,0,334);
-INSERT INTO `mob_pools` VALUES (6086,'Djokvukks_Wyvern','Djokvukks_Wyvern',193,0x0000F90200000000000000000000000000000000,14,14,7,200,100,0,1,0,1,0,0,32,0,0,0,0,0,0,0,714);
+INSERT INTO `mob_pools` VALUES (6086,'Djokvukks_Wyvern','Djokvukks_Wyvern',193,0x0000F90200000000000000000000000000000000,14,14,7,200,100,0,1,0,1,0,0,32,0,155,0,0,0,0,0,714);
 INSERT INTO `mob_pools` VALUES (6087,'Arch_Overlord_Tombstone','Arch_Overlord_Tombstone',93,0x0000210400000000000000000000000000000000,5,4,11,240,100,0,1,1,1,2,1023,0,6144,129,5,0,49,0,0,93);
 INSERT INTO `mob_pools` VALUES (6088,'ZoPha_Forgesoul','ZoPha_Forgesoul',337,0x00001F0400000000000000000000000000000000,4,12,4,265,100,0,1,1,1,2,0,32,7,159,0,0,2,1,0,705);
 INSERT INTO `mob_pools` VALUES (6089,'Fuu_Tzapo_the_Blessed','Fuu_Tzapo_the_Blessed',360,0x00003C0400000000000000000000000000000000,3,1,11,240,100,0,1,1,1,2,0,32,1,159,0,0,1,1,0,710);
 INSERT INTO `mob_pools` VALUES (6090,'Naa_Yixo_the_Stillrage','Naa_Yixo_the_Stillrage',360,0x00003A0400000000000000000000000000000000,5,5,2,240,100,0,1,1,1,2,0,32,7,159,0,0,3,1,0,710);
 INSERT INTO `mob_pools` VALUES (6091,'Tee_Zaksa_the_Ceaseless','Tee_Zaksa_the_Ceaseless',360,0x00003B0400000000000000000000000000000000,7,7,10,240,100,0,1,1,1,2,0,32,1,159,0,0,4,1,0,710);
 INSERT INTO `mob_pools` VALUES (6092,'Arch_Tzee_Xicu_Idol','Arch_Tzee_Xicu_Idol',95,0x0000250400000000000000000000000000000000,4,4,12,240,100,0,1,1,1,2,1023,0,0,159,0,0,50,1,0,95);
-INSERT INTO `mob_pools` VALUES (6093,'Angra_Mainyu','Angra_Mainyu',4,0x0000090100000000000000000000000000000000,5,4,2,240,100,0,1,1,1,2,513,32,7,159,0,1,0,1,0,4);
+INSERT INTO `mob_pools` VALUES (6093,'Arch_Angra_Mainyu','Arch_Angra_Mainyu',4,0x0000090100000000000000000000000000000000,5,4,2,240,100,0,1,1,1,2,513,32,7,159,0,1,0,1,0,4);
```

Comment:
Both Angra Mainyu and Arch Angra Mainyu exist with Neo-Dynamis, I think you'll probably need 2 groups?

Reply:
There exist separate group/pool for the regular megaboss.

Reply:
That's what I get for only looking at the diff.